test/e2e/network/no_snat.go

```diff
@@ -40,8 +40,8 @@ const (
 	testProxyPort = 31235 // Firewall rule allows external traffic on ports 30000-32767. I just picked a random one.
 )
 
-var testPodImage = imageutils.GetE2EImage(imageutils.NoSnatTest)
-var testProxyImage = imageutils.GetE2EImage(imageutils.NoSnatTestProxy)
+var testPodImage = imageutils.GetE2EImage(imageutils.Agnhost)
+var testProxyImage = imageutils.GetE2EImage(imageutils.Agnhost)
```

Comment:
nit: these don't seem to be re-used elsewhere, I find it more legible to have the `imageutils` decl right next to the args being used

Reply:
makes sense.